pages/api/admin/users/readVerified.ts

```diff
@@ -0,0 +1,24 @@
+import { PrismaClient } from "@prisma/client";
+import { NextApiResponse } from "next";
+import sanitizeUser from "utils/sanitizeUser";
+
+const prisma = new PrismaClient();
+
+export default async (res: NextApiResponse): Promise<void> => {
+  try {
+    const user = await prisma.user.findMany({
+      where: { emailVerified: null },
+    });
+
+    if (!user) {
+      res
+        .status(404)
+        .json({ statusCode: 404, message: "User does not exist." });
```

Comment:
This status code shouldn't be a 404 because if we find there are no users where the email hasn't been verified, there is no error - it just means that there are no users who have requested making an account. We should be returning a 204 response saying that there are no user account requests

Reply:
Yeah, @christopher-grey, I don't think you actually need to handle this case because `user` will always be truthy (empty arrays are the empty response here, not null). Might make more sense to call the variable `users`, since the type will be `User[]`.